### ISSUE TITLE
last.survey.year is still 2017

### ISSUE THREAD

**Issue by andrew-edwards** [pacific-hake/hake-assessment#606]
It comes from
```
last.survey.year <- survey.history[nrow(survey.history),]$year
```
so need to dig further.

**Comment by andrew-edwards** [comment]
```
> survey.history[,1:3]
   year start.date end.date
1  1995     01-Jul   01-Sep
2  1998     06-Jul   27-Aug
3  2001     15-Jun   18-Aug
4  2003     29-Jun   01-Sep
5  2005     20-Jun   19-Aug
6  2007     20-Jun   21-Aug
7  2009     30-Jun   07-Sep
8  2011     26-Jun   10-Sep
9  2012     23-Jun   07-Sep
10 2013     13-Jun   11-Sep
11 2015     15-Jun   14-Sep
12 2017     22-Jun   13-Sep
```

**Comment by cgrandin** [comment]
Fixed in a5051bf04346d995bbd507425a06b88b694096dd
